app/pods/controller-injections-1/parent-component.js

```diff
@@ -12,10 +12,10 @@ export default Component.extend({
       Parent data: {{this.data}}
     </p>
 
-    {{yield data}}
+    {{yield 4 5}}
```

Comment:
Oops, now you're passing static value `4`  as data, but you need to pass the value of the computed property! 😅

Reply:
i know, but i only wanted to grasp the comopnents communication way. thank you